README.md

```diff
@@ -82,33 +91,52 @@ roslaunch mqtt_client standalone.launch
-[ INFO] [1652888440.062255501]: Connected to broker at 'tcp://localhost:1883'
+[ WARN] [1665575657.358869079]: Parameter 'broker/tls/enabled' not set, defaulting to '0'
+[ WARN] [1665575657.359798329]: Parameter 'client/id' not set, defaulting to ''
+[ WARN] [1665575657.359810889]: Client buffer can not be enabled when client ID is empty
+[ WARN] [1665575657.360300703]: Parameter 'client/clean_session' not set, defaulting to '1'
+[ WARN] [1665575657.360576344]: Parameter 'client/keep_alive_interval' not set, defaulting to '60.000000'
+[ WARN] [1665575657.360847295]: Parameter 'client/max_inflight' not set, defaulting to '65535'
+[ INFO] [1665575657.361281461]: Bridging ROS topic '/ping/ros' to MQTT topic 'pingpong/ros' 
+[ INFO] [1665575657.361303380]: Bridging primitive ROS topic '/ping/primitive' to MQTT topic 'pingpong/primitive' 
+[ INFO] [1665575657.361352809]: Bridging MQTT topic 'pingpong/ros' to ROS topic '/pong/ros'
+[ INFO] [1665575657.361370558]: Bridging MQTT topic 'pingpong/primitive' to primitive ROS topic '/pong/primitive'
+[ INFO] [1665575657.362153083]: Connecting to broker at 'tcp://localhost:1883' ...
+[ INFO] [1665575657.462622065]: Connected to broker at 'tcp://localhost:1883'
 ```
 
 Note that the *mqtt_client* successfully connected to the broker and also echoed which ROS/MQTT topics are being bridged.
 
-In order to test the communication, publish any message on ROS topic `/ping` and wait for a response on ROS topic `/pong`. To this end, open two new terminals and execute the following commands.
+In order to test the communication among *mqtt_clients*, publish any ROS message on ROS topic `/ping/ros` and wait for a response on ROS topic `/pong/ros`.
 
 ```bash
-# 1st terminal: listen on /pong
-rostopic echo /pong
+# 1st terminal: listen for ROS messages on /pong/ros
+rostopic echo /pong/ros
 ```
 
 ```bash
-# 2nd terminal: publish to /ping
-rostopic pub -r 1 /ping std_msgs/String "Hello MQTT!"
+# 2nd terminal: publish ROS message to /ping/ros
+rostopic pub -r 1 /ping/ros std_msgs/String "Hello MQTT!"
 ```
 
-If everything works as expected, a new message should be printed in the first terminal once a second.
+In order to test the communication between *mqtt_client* and other MQTT clients, publish a primitive ROS message on ROS topic `/ping/primitive`, directly publish a primitive MQTT message on MQTT topic `pingpong/primitive` and wait for responses on ROS topic `/pong/primitive`.
+
+```bash
+# 3rd terminal: listen for primitive ROS messages on /pong/primitive
+rostopic echo /pong/primitive
+```
+
+```bash
+# 4th terminal: publish primitive ROS message to /ping/primitive
+rostopic pub -r 1 /ping/primitive std_msgs/Int32 42
+```
+
+```bash
+# 5th terminal: publish primitive MQTT message to pingpong/primitive
+docker run --rm --network host eclipse-mosquitto mosquitto_pub -h localhost -t "pingpong/primitive" --repeat 10 --repeat-delay 1 -m 69
```

Comment:
ROS messages are sent indefinitely, primitive messages are only sent 10 times. I did not find a good way to send the primitive messages indefinitely but maybe we can increase this to 100 to give enough time to find the correct terminal again ;-)

```suggestion
docker run --rm --network host eclipse-mosquitto mosquitto_pub -h localhost -t "pingpong/primitive" --repeat 100 --repeat-delay 1 -m 69
```